Desktop Version

MarcoOquendoC opened the pull request:
# [Portfolio: desktop version: day 2](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)

## Hi good reviewer,

### These are the changes proposed to the Portfolio project:

#### index.html
- Change **class names** to summarize tag contents. 
- Erase useless **div** tags. 
- Restructure the whole project.
#### styles.css
- Change **class names** to correspond with the same element in HTML.
- Erase **repetitive** style attributes.
- Implement responsive grid design (without media queries).
- Add media queries to complete responsive design. 
- Modify animation to new name and behavior to "rotate-and-color". It modifies the subtitle ".ocupation".

#### Project requirements
> Ignore the differentiation of First and Last names in the desktop version seen in [Figma template 3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988).
> Use media queries in your CSS file to make the desktop version look like the selected template (Desktop: 768px or wider).
> To lay out the elements on the page use Grid and Flexbox.
> Replicate the design as much as possible (e.g., font, colors, images, text, margins) using the info from [Figma template 3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988).
> Collaborate with coding partner [Addis](https://github.com/Aseadd).
> Collaborate with coding partner [Ademola](https://github.com/Ademola-coding).

![Thank you](https://user-images.githubusercontent.com/104022866/178110408-9de8a29a-c74b-44a4-b0dd-04ab3ed9a1e5.gif)

 I liked a lot learning animation. Do you have any pro tips on that subject?
Thanks, beforehand.

Aseadd committed:
Animation in subtitle

MarcoOquendoC committed:
Change class names to summarize tag contents. Erase useless div tags. Reestructure all project.

MarcoOquendoC committed:
Change class names to correspond with the same element in html. Add media queries and responsive grid design. Erase animation to implement it in a new commit.

MarcoOquendoC committed:
Add animation rotate and color to subtitle o cupation.

prolajumokeoni commented on index.html:
```diff
@@ -1,225 +1,240 @@
+</head>
+<body>
+  <header>
+    <span class="welcome">Welcome</span>
+    <nav>
+      <a href="#"><img src="/images/Icon-Menu.png" alt="hamburger-icon"></a>
+    </nav>
```
- [ ] For desktop navigation, the logo and hamburger button shouldn't be displayed in this screen size

- [ ] Kindly add and center the links 👍🏿 


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_320](https://user-images.githubusercontent.com/69638013/178111677-60efb9b7-965c-4174-82f9-c30ce1ead4a6.png) | ![Screenshot_319](https://user-images.githubusercontent.com/69638013/178111579-be768fa5-e46e-404e-a5ed-24521cbd42ae.png)

prolajumokeoni commented on index.html:
```diff
@@ -1,225 +1,240 @@
+  </header>
+  <main>
+    <section class="headline">
+      <h1 class="title name">Hey there. I'm Marco</h1>
+      <h2 class="title ocupation">I'm a software developer</h2>
+      <p class="description">
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <div class="social-container">
+        <ul>
+          <li><a href="#"><img src="/images/twitter.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/Linkedin-icon.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/medium.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/github.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/angellist.png" alt=""></a></li>
+        </ul>
+      </div>
```
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 3-main](https://user-images.githubusercontent.com/69638013/178111919-646b6940-1078-413b-93f5-b855fcba8f21.png)  |  ![Screenshot_321](https://user-images.githubusercontent.com/69638013/178112079-8bbc0057-6f10-44db-96ed-79709a92935e.png)


- [ ] Kindly ensure background image is visible on desktop 👍🏿 
##OPTIONAL
- [ ]  Kindly add alt text to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible

prolajumokeoni commented on index.html:
```diff
@@ -1,225 +1,240 @@
+        My Recent Works
+      </h2>
+      <hr>
+      <div class="work-grid">
```
- [ ] Kindly ensure that your cards are in 2 rows with 3 projects each (6 projects in total), currently this is only achieved in `1440px` screensize and above, please make sure that screensizes from `769px` where your media query starts has this implemented 👍🏿

prolajumokeoni commented on index.html:
```diff
@@ -1,225 +1,240 @@
+          Get My Resume
+        </button>
+      </div>
+      <div class="about-grid">
```
- [ ] Kindly align the cards in this section from `769px` to be 3 on 1 row, just like in the figma design, see video below to see how your website looks from my end 👍🏿 

https://user-images.githubusercontent.com/69638013/178112797-4ac05bac-d6d2-44fd-8d20-af2dad3f4cd6.mp4

prolajumokeoni commented on styles.css:
```diff
@@ -315,29 +352,48 @@ body {
+  }
+}
+
+@media only screen and (max-width: 768px) {
```
- [ ] Kindly use `min-width` instead of `max-width`, currently this is affecting your background image from being displayed on desktop

prolajumokeoni reviewed (changes requested):
Hi @MarcoOquendoC, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MarcoOquendoC committed:
Add missing images for different backgrounds

MarcoOquendoC committed:
Modify index.html.
Add alt text to img tags.
Add top links.
Add id attribute for navigation from top links.

MarcoOquendoC committed:
Modify styles.css
Add styles for top navigation.
Modify animation to only change color.
Fix grid template columns for card containers.
Modify media queries to hide hamburguer icon on big screens

MarcoOquendoC committed:
Fix linter errors

MarcoOquendoC commented:
# Hi @prolajumokeoni,
## Thank you for your review.

### The changes on the repository are listed below:

- Hide hamburger icon for desktop navigation.
- Add and center the links. Also, they can be click to navigate the webpage.
- Ensure the background image is visible on desktop templates (the image was not in the correct folder).
- Add alt text to all images.
- Modify the grid template columns for card containers to show cards on screens (width sizes over 768px).
- Modify media query to use min-width instead of max-width. This change was not necessary since the issue was the missing background image file in the images folder.

![Have a good day bear](https://user-images.githubusercontent.com/104022866/178123015-020aa258-2a43-4303-a0eb-36f15d27c526.gif)

1GORDON commented on styles.css:
```diff
@@ -29,58 +45,51 @@ header {
 }
 
 .ocupation {
-  color: #36b37e;
+  animation: color 2s ease infinite alternate;
 }
```
Great job @MarcoOquendoC 🤝

- [ ] [Optional] You may please remove this animation. It is not a requirement at the moment and it makes your design different from the Figma template that was provided in the project requirements.

1GORDON commented on styles.css:
```diff
@@ -91,192 +100,218 @@ li {
+
+.work-grid {
   display: grid;
-  grid-template-columns: 1fr;
-  row-gap: 24px;
+  grid-auto-rows: auto;
+  grid-template-columns: repeat(auto-fill, minmax(min(100%, 13rem), 1fr));
+  gap: 1vw;
+  margin: 2vw;
 }
```
Great work @MarcoOquendoC 🤝

- [ ] Just like the previous reviewer suggested. You may please make sure your projects appear in two rows just like in the Figma template provided
- Your design
![your projects](https://user-images.githubusercontent.com/63917725/178123625-2eee368a-92dc-4cfb-b729-542348e4aba1.PNG)

- Figma design
![Figma projects](https://user-images.githubusercontent.com/63917725/178123649-660c60cc-d5d3-484c-a008-ecb207068165.PNG)

1GORDON reviewed (changes requested):
##
## Hi @MarcoOquendoC ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Readme ✔️
- Linters are passing ✔️
- Professional Pull Request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] [Optional] You may kindly work on your design properly. It still seems to be very different from the design provided in the project requirements. Kindly please make your design at least close to the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MarcoOquendoC committed:
Modify styles.css. Remove animation. Add media query to show three columns in card container on desktop version.

MarcoOquendoC commented:
# Hi @1GORDON,

## Thanks for your kindness

### These are the new changes in the repository:

- Make sure projects appear in two rows on a desktop version.
- Remove animation.

![Sir, yes, sir](https://user-images.githubusercontent.com/104022866/178124426-e4f6b17e-37f0-4ac0-b98f-ec324cc796ad.gif)

tufoinnkuo10 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @MarcoOquendoC 

Your project is complete! Great job for making all the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MarcoOquendoC commented:
Thank you @tufoinnkuo10! 😁😍
I love the word "Approved".
Have a nice sunday.
Marco.
